shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -765,6 +848,13 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
         !node.HasFlag(flutter::SemanticsFlags::kIsReadOnly)) {
       // Text fields are backed by objects that implement UITextInput.
       object = [[[TextInputSemanticsObject alloc] initWithBridge:GetWeakPtr() uid:uid] autorelease];
+    } else if (node.HasFlag(flutter::SemanticsFlags::kHasToggledState) ||
```

Comment:
You'll probably also have to add some code below to handle the case where a SemanticsNode switches from having these flags to not having them and vice versa.

Reply:
Can that happen that a node will have kHasToggledState and then it doesn't?  That seems like it would be immutable in practice since a new SemanticsObject would get generated for a new entity that is no longer a switch.  Are we recycling SemanticsObjects?

Reply:
While unlikely, the framework supports switching the role of a SemanticsObject, that's why we have the code below for text fields.

Reply:
Ok, I think it's pretty weird to accommodate for something that shouldn't happen, but done.